app/src/main/java/com/shagalalab/constitution/data/ConstitutionDatabase.kt

```diff
@@ -0,0 +1,39 @@
+package com.shagalalab.constitution.data
+
+import android.content.Context
+import androidx.room.Database
+import androidx.room.Room
+import androidx.room.RoomDatabase
+import com.shagalalab.constitution.data.dao.ArticleDao
+import com.shagalalab.constitution.data.dao.ChapterDao
+import com.shagalalab.constitution.data.dao.PartDao
+import com.shagalalab.constitution.data.models.ArticleModel
+import com.shagalalab.constitution.data.models.ChapterModel
+import com.shagalalab.constitution.data.models.PartModel
+
+@Database(
+    entities = [ArticleModel::class, ChapterModel::class, PartModel::class],
+    version = 1,
+    exportSchema = false
+)
+abstract class ConstitutionDatabase : RoomDatabase() {
+
+    abstract fun articleDao(): ArticleDao
+    abstract fun chapterDao(): ChapterDao
+    abstract fun partDao(): PartDao
+
+    companion object {
+        private lateinit var INSTANCE: ConstitutionDatabase
+        fun getInstance(context: Context): ConstitutionDatabase {
+            INSTANCE = Room.databaseBuilder(
```

Comment:
Your previous implementation was almost correct (just needed minor change). Now here you are creating `ConstitutionDatabase` instance every time we call the method. We need to create `ConstitutionDatabase` only once, when we call next time it should return previously created one (this pattern is called `Singleton`).